### PR TITLE
Update participant postal code (and also FSA)

### PR DESCRIPTION
**Pull request by dbayly-freshworks**

### CODE REVIEW

File: client/src/components/modal-forms/EditParticipantForm.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ export const EditParticipantForm = ({ initialValues, validationSchema, onSubmit,
               label='* Email Address'
               type='email'
             />
+            <Field name='postalCode' component={RenderTextField} label='Postal Code' type='text' />
```
Comment (Zenen-freshworks) [inline client/src/components/modal-forms/EditParticipantForm.js:28]: nitpick: inconsistent formatting with surrounding fields

**Review by Zenen-freshworks — APPROVED** [review]
Everything looks good except for the formatting nitpick

File: client/src/components/modal-forms/EditParticipantForm.js (hunk 1 above)
Comment (arranfw) [reply client/src/components/modal-forms/EditParticipantForm.js:28]: I think this it the right way, prettier breaks components down once they reach a certain line length.

Comment (Zenen-freshworks) [reply client/src/components/modal-forms/EditParticipantForm.js:28]: it's the `prettier` way, but is it the _prettier_ way? 🤔

Comment (arranfw) [reply client/src/components/modal-forms/EditParticipantForm.js:28]: Praise be to prettier, heresy begone

**Review by arranfw — APPROVED** [review]
Looks good, tested locally and distance updates as expected

File: client/src/components/modal-forms/EditParticipantForm.js (hunk 1 above)
Comment (Zenen-freshworks) [reply client/src/components/modal-forms/EditParticipantForm.js:28]: your dogma will be our undoing

**Pull request merged**